content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/300-next-steps.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Next steps'
+metaTitle: 'Next steps'
+metaDescription: 'Next steps to take now that you have successfully added Prisma to your project'
+langSwitcher: ['typescript', 'node']
+dbSwitcher: ['mongodb']
+toc: false
+---
+
+## Next steps
+
+This section lists a number of potential next steps you can now take from here. Feel free to explore these or read the [Introduction](../../../../concepts/overview/what-is-prisma) page to get a high-level overview of Prisma.
+
+### Continue exploring the Prisma Client API
+
+You can send a variety of queries with the Prisma Client API. Check out the [API reference](../../../../concepts/components/prisma-client) and use your existing database setup from this guide to try them out.
+
+<Tip>
+
+You can use your editor's auto-completion feature to learn about the different API calls and the arguments it takes. Auto-completion is commonly invoked by hitting <kbd>CTRL</kbd>+<kbd>SPACE</kbd> on your keyboard.
+
+</Tip>
+
+<details><summary>Expand for more Prisma Client API examples</summary>
+
+Here are a few suggestions for a number of more queries you can send with Prisma Client:
+
+**Filter all `Post` records that contain `"hello"`**
+
+```js
+const filteredPosts = await prisma.post.findMany({
+  where: {
+    OR: [{ title: { contains: 'hello' } }, { content: { contains: 'hello' } }],
+  },
+})
+```
+
+**Create a new `Post` record and connect it to an existing `User` record**
+
+```js
+const post = await prisma.post.create({
+  data: {
+    title: 'Join us for Prisma Day 2020',
+    author: {
+      connect: { email: 'alice@prisma.io' },
+    },
+  },
+})
+```
+
+**Use the fluent relations API to retrieve the `Post` records of a `User` by traversing the relations**
+
+```js
+const posts = await prisma.profile
+  .findUnique({
+    where: { id: 1 },
+  })
+  .user()
+  .posts()
+```
+
+**Delete a `User` record**
+
+```js
+const deletedUser = await prisma.user.delete({
+  where: { email: 'sarah@prisma.io' },
+})
+```
+
+</details>
+
+### Explore the data in Prisma Studio
+
+Prisma Studio is a visual editor for the data in your database.
+You can run it with two ways:
+
+1. Run `$ npx prisma studio` in your terminal.
+2. Install [the desktop app](https://github.com/prisma/studio/releases) from the installers. Windows, macOS and Linux are supported.
```

Comment:
No problems to expect with Studio and Mongo I guess?

Reply:
Not unless it uses raw SQL - will check